target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -199,6 +199,9 @@ hc5861)
 	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wlan5g" "rai0"
 	ucidef_set_led_default "turbo" "turbo" "$board:blue:turbo" "0"
 	;;
+hc5962)
+	ucidef_set_led_timer "status" "status" "$board:white:status" "1000" "1000"
```

Comment:
Hi.  
This device has only one indicator light, So, After the system starts, I hope the Indicator blink white to show system normal operation.
![Image of Yaktocat](http://www.hiwifi.com/static/img/j4t-b2.jpg)